components/frontend/src/source/Source.js

```diff
@@ -68,15 +68,17 @@ function Parameters({ changed_fields, config_error, connection_error, metric, pa
                     />
                 </Grid.Column>
             </Grid.Row>
-            <Grid.Row columns={2}>
```

Comment:
Was it incorrectly setting 2 columns before?

Reply:
No, it was correct. In the previous version, the parameters would be laid out in two columns because of this setting. In the current version, the `SourceParameters` component does the layout across columns.